src/for_3D_build/particle_dynamics/solid_dynamics/slender_structure_dynamics.h

```diff
@@ -231,14 +231,14 @@ class BarStressRelaxationFirstHalf : public BaseBarRelaxation
 
         force_[index_i] = force * inv_rho0_ / (thickness_[index_i] * width_[index_i]);
         dpseudo_n_d2t_[index_i] = pseudo_normal_acceleration * inv_rho0_ * 12.0 / pow(thickness_[index_i], 4);
-        dpseudo_b_n_d2t_[index_i] = -pseudo_b_normal_acceleration * inv_rho0_ * 12.0 / pow(thickness_[index_i], 4);
+        dpseudo_b_n_d2t_[index_i] = pseudo_b_normal_acceleration * inv_rho0_ * 12.0 / pow(thickness_[index_i], 4);
 
         Vecd local_dpseudo_n_d2t = transformation_matrix_[index_i] * dpseudo_n_d2t_[index_i];
         Vecd local_dpseudo_b_n_d2t = transformation_matrix_[index_i] * dpseudo_b_n_d2t_[index_i];
-        dangular_b_vel_dt_[index_i] = getRotationFromPseudoNormalForSmallDeformation_b(
-            Vec3d(local_dpseudo_b_n_d2t), Vec3d(local_dpseudo_n_d2t), Vec3d(rotation_b_[index_i]), Vec3d(angular_b_vel_[index_i]), dt);
-        dangular_vel_dt_[index_i] = getRotationFromPseudoNormalForSmallDeformation(
-            Vec3d(local_dpseudo_b_n_d2t), Vec3d(local_dpseudo_n_d2t), Vec3d(rotation_[index_i]), Vec3d(angular_vel_[index_i]), dt);
+        dangular_b_vel_dt_[index_i] = getRotationFromPseudoNormalForFiniteDeformation_b(
```

Comment:
Since this pull request focuses on Finite Deformation/Rotation, could you add a test case with a large rotation?